hack/jenkins/release_build_and_upload.sh

```diff
@@ -111,5 +112,20 @@ fi
 #echo "Updating Docker images ..."
 #make push-gvisor-addon-image push-storage-provisioner-manifest
 
+echo "Generating tarballs for kicbase images"
+# first get the correct tag of the kic base image
+KIC_VERSION=$(grep -E "Version =" pkg/drivers/kic/types.go | cut -d \" -f 2 | cut -d "-" -f 1)
+# then generate tarballs for all achitectures
+for ARCH in "amd64" "arm64" "arm/v7" "ppc64le" "s390x" 
+do
+  SUFFIX=$(echo $ARCH | sed 's/\///g')
+  IMAGE_NAME=kicbase/stable:${KIC_VERSION}
```

Comment:
can we ensure that this image is already pushed into the registry ...otherwise it wont find it...(maybe it is somewhere up in the script that pushes the image)

Reply:
@spowelljr is this the correct place to generate the tarball for new kicbase image? I guess this is a part of jenkins job, and the docker image should have been pushed into registry before this?